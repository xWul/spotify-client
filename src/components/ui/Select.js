import React from 'react'
import styled from 'styled-components'

const Content = styled.div`
  width: 100%;
  padding: 5px;
`

const SelectStyled = styled.select`
  background-color: #FFF;
  cursor: pointer;
  padding: 5px;
  width: 100%;
  border: 1px solid #333;
  border-radius: 3px;
  font-size: 18px;
`

const Select = (props) => {
  return (
    <Content>
      <SelectStyled {...props} />
    </Content>
  )
}

export default Select
