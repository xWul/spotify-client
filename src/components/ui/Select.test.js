import React from 'react'
import renderer from 'react-test-renderer'
import Select from './Select'

test('Select renderer without crash', () => {
  const component = renderer.create(
    <Select>
      <option value={'1'}>1</option>
      <option value={'2'}>2</option>
    </Select>,
  )

  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
