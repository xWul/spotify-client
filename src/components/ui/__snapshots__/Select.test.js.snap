// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select renderer without crash 1`] = `
<div
  className="sc-bdVaJa LDKxg"
>
  <select
    className="sc-bwzfXH fGrHko"
  >
    <option
      value="1"
    >
      1
    </option>
    <option
      value="2"
    >
      2
    </option>
  </select>
</div>
`;
